skills_ml/algorithms/occupation_classifiers/base.py

```diff
@@ -63,51 +62,64 @@ def _load_model(self):
         Returns:
             gensim.models.doc2vec.Doc2Vec: The word-embedding model object.
         """
-        if not self.saved:
-            with tempfile.TemporaryDirectory() as td:
-                for f in self.files:
-                    filepath = os.path.join(td, f)
-                    if not os.path.exists(filepath):
+        try:
+            model = Doc2Vec.load('tmp/' + self.model_name)
```

Comment:
I think using a hardcoded 'tmp' directory is getting hard to manage. Can we pass the 'local  cache directory' value in, and ultimately control it through config?